data/api/v1/translate_actions.json

```diff
@@ -938,4 +938,4 @@
     "description": "Disables muting all monitors. Throws an error if mute all\nwas not enabled previously.",
     "summary": "Unmute all monitors"
   }
-}
\ No newline at end of file
+}
```

Comment:
I think this page has to be changed through the API spec repo.

Reply:
Do you mind pointing me where this is? Happy to make the changes there but couldn't find it in the documentation repo.

Reply:
@NBParis https://github.com/DataDog/datadog-api-spec/blob/master/spec/v1/monitor.yaml

Reply:
Opened a PR and will check with engineering on making it through: https://github.com/DataDog/datadog-api-spec/pull/698